yearn/deploy/00_deploy_cvp_strategy.ts

```diff
@@ -0,0 +1,49 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts, helpers } = hre
+  const { deploy, log } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  const TBTCCurveVault = await deployments.getOrNull("TBTCCurveVault")
+  if (!TBTCCurveVault || !helpers.address.isValid(TBTCCurveVault.address)) {
+    throw new Error("Address of TBTCCurveVault must be set")
+  }
+
+  const TBTCCurvePoolDepositor = await deployments.getOrNull("TBTCCurvePoolDepositor")
+  if (!TBTCCurvePoolDepositor || !helpers.address.isValid(TBTCCurvePoolDepositor.address)) {
+    throw new Error("Address of TBTCCurvePoolDepositor must be set")
+  }
+
+  const TBTCCurvePoolGauge = await deployments.getOrNull("TBTCCurvePoolGauge")
+  if (!TBTCCurvePoolGauge || !helpers.address.isValid(TBTCCurvePoolGauge.address)) {
+    throw new Error("Address of TBTCCurvePoolGauge must be set")
+  }
+
+  const TBTCCurvePoolGaugeReward = await deployments.getOrNull("TBTCCurvePoolGaugeReward")
+  if (!TBTCCurvePoolGaugeReward || !helpers.address.isValid(TBTCCurvePoolGaugeReward.address)) {
+    TBTCCurvePoolGaugeReward.address = "0x0000000000000000000000000000000000000000"
+  }
+
+  log(`tbtcCurveVault: ${TBTCCurveVault.address}`)
+  log(`tbtcCurvePoolDepositor: ${TBTCCurvePoolDepositor.address}`)
+  log(`tbtcCurvePoolGauge: ${TBTCCurvePoolGauge.address}`)
+  log(`tbtcCurvePoolGaugeReward: ${TBTCCurvePoolGaugeReward.address}`)
```

Comment:
```suggestion
  log(`TBTCCurveVault: ${TBTCCurveVault.address}`)
  log(`TBTCCurvePoolDepositor: ${TBTCCurvePoolDepositor.address}`)
  log(`TBTCCurvePoolGauge: ${TBTCCurvePoolGauge.address}`)
  log(`TBTCCurvePoolGaugeReward: ${TBTCCurvePoolGaugeReward.address}`)
```

Reply:
Those are constructor parameters that are uncapitalized in the contract so this is why I display them that way to avoid confusion. I'd rather prefer to leave as is but arguments for making them capitalized are welcome :)